<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Keyboard</class>
 <widget class="QWidget" name="Keyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyboard</string>
  </property>
  <widget class="QPushButton" name="number_button_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_13">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_24">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="number_button_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(26, 95, 180)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_14">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(26, 95, 180)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(26, 95, 180)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_25">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(26, 95, 180)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="shift_button_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>⇧</string>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_26">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(6, 234, 2)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="number_button_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(6, 234, 2)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(6, 234, 2)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_15">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(6, 234, 2)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_16">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(192, 28, 40)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="number_button_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(192, 28, 40)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_27">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(192, 28, 40)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(192, 28, 40)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_17">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(192, 28, 40)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_28">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(192, 28, 40)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="number_button_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(192, 28, 40)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(192, 28, 40)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_29">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(129, 61, 156)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_6">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(129, 61, 156)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="number_button_6">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(129, 61, 156)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_18">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(129, 61, 156)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_32">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(26, 95, 180)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_33">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_8">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(6, 234, 2)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_30">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(129, 61, 156)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="number_button_10">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_19">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(129, 61, 156)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_9">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(26, 95, 180)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="number_button_11">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_20">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(6, 234, 2)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_12">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_22">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_31">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(6, 234, 2)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_23">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_10">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_7">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(129, 61, 156)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="number_button_8">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(6, 234, 2)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="number_button_9">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(26, 95, 180)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="number_button_12">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_21">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>210</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(26, 95, 180)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="symbol_button_11">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>110</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="number_button_7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(129, 61, 156)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="shift_button_2">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>310</y>
     <width>140</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>⇧</string>
   </property>
  </widget>
  <widget class="QPushButton" name="space_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>480</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="extra_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>40</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(245, 194, 17)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
